File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/Button/Functionality/CustomDialogForm.vb
Imports Gizmox.WebGUI.Forms
Imports Gizmox.WebGUI.Common

Public Class CustomDialogForm
    Public Sub New()
        InitializeComponent()
        'Applying DialogResult property for "Yes" and "No" buttons
        Me.mobjNoButton.DialogResult = Gizmox.WebGUI.Forms.DialogResult.No
        Me.mobjYesButton.DialogResult = Gizmox.WebGUI.Forms.DialogResult.Yes
    End Sub
End Class

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/Button/Functionality/CustomDialogForm.cs
#region Using

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;

using Gizmox.WebGUI.Common;
using Gizmox.WebGUI.Forms;

#endregion

namespace CompanionKit.Controls.Button.Functionality
{
    public partial class CustomDialogForm : global::Gizmox.WebGUI.Forms.Form
    {
        public CustomDialogForm()
        {
            InitializeComponent();
            //Applying DialogResult property for "Yes" and "No" buttons
            this.mobjNoButton.DialogResult = Gizmox.WebGUI.Forms.DialogResult.No;
            this.mobjYesButton.DialogResult = Gizmox.WebGUI.Forms.DialogResult.Yes;
        }
    }
}